hooks/terraform_validate.sh

```diff
@@ -100,6 +101,7 @@ function per_dir_hook_unique_part {
 
     case $key in
       --retry-once-with-cleanup)
+        # shellcheck disable=SC2086 # It's just works. Maybe will be fixed later
```

Comment:
```suggestion
        # shellcheck disable=SC2086 # It just works. Maybe will be fixed later
```
JFYI: it should work the same way if the var is wrapped into double quotes (`if [ "$retry_once_with_cleanup" ]; then`), though as suggested by `shellcheck` is a way much more secure.

Reply:
Hmm, okay. Let's then add quotes, and hope that it will not break anything (sometime I'll wrote tests, but not today)